### PR TITLE
Update the max leverage button to 25

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
Update the max leverage button to 25x

## Motivation and Context
Improve the UX

## How Has This Been Tested?
1. Select ETH market on optimism kovan and check the max leverage button is 25x
2. Select XAU market on optimism kovan and check the max leverage button is 10x

## Screenshots (if appropriate):
image
image

### CODE REVIEW

File: sections/futures/LeverageInput/LeverageInput.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const SliderRow = styled(FlexDivRow)`
 
 const LeverageInputContainer = styled.div`
 	display: grid;
-	grid-template-columns: 1fr 43px 43px 43px;
+	grid-template-columns: 1fr 43px 43px 43px 43px;
```
[the remark this answers is not in the document]
Comment (LeifuChen) [reply sections/futures/LeverageInput/LeverageInput.tsx:146]: Sure. Let me change it.

File: store/futures/index.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { zeroBN } from 'utils/formatters/number';
 import { Synths, CurrencyKey } from 'constants/currency';
 import { DEFAULT_NP_LEVERAGE_ADJUSTMENT } from 'constants/defaults';
 
-const DEFAULT_MAX_LEVERAGE = wei(10);
+const DEFAULT_MAX_LEVERAGE = wei(25);
```
[the remark this answers is not in the document]
Comment (LeifuChen) [reply store/futures/index.ts:16]: Got it.

File: sections/futures/LeverageInput/LeverageInput.tsx [hunk 3]
```diff
@@ -50,6 +50,7 @@ const LeverageInput: FC<LeverageInputProps> = ({ onLeverageChange }) => {
 		return position?.remainingMargin.lte(0) || maxLeverage.lte(0);
 	}, [position, maxLeverage]);
 
+	const leverageButtons = maxLeverage.eq(25) ? ['5', '10', '25'] : ['2', '5', '10'];
```
Comment (koredefashokun) [inline sections/futures/LeverageInput/LeverageInput.tsx:53]: I think the maxLeverage value changes when positions are open, right? So handling those cases might need us to do things that are trickier than this.

Comment (LeifuChen) [reply sections/futures/LeverageInput/LeverageInput.tsx:53]: The value of `maxLeverage` is handled here https://github.com/Kwenta/kwenta/blob/dev/store/futures/index.ts#L108

**Review by Tburm — APPROVED** [review]

**Pull request merged**